.travis.yml

```diff
@@ -1,59 +1,64 @@
 language: ruby
+sudo: required
+services:
+  - docker
 cache:
   bundler: true
   yarn: true
   directories:
-    - node_modules
-    - public/assets
-    - public/packs-test
-    - tmp/cache/babel-loader
+  - node_modules
+  - public/assets
+  - public/packs-test
+  - tmp/cache/babel-loader
 dist: trusty
 sudo: false
 branches:
   only:
-    - master
-
+  - master
+  - /^v\d+\.\d+(\.\d+)?(\S*)?(-\S*)?$/
```

Comment:
Can you clarify what this regex is?

Reply:
It matches branches (yes, for some reason Travis detects tags as branches) with release criteria, i.e. "v2.3.0rc1", or "v.2.3.4-test".